Source/Scene/Cesium3DTileBatchTable.js

```diff
@@ -132,6 +134,21 @@ define([
         this._textureStep = textureStep;
     }
 
+    defineProperties(Cesium3DTileBatchTable.prototype, {
+        textureMemoryInBytes : {
```

Comment:
Ah, I did not think about batch table vs. texture memory.  Perhaps it is better to report these separately so it is less confusing to the end users that might not realize that the batch table is stored in a texture.